.travis.yml

```diff
@@ -1,25 +1,6 @@
 language: python
 python:
   - 2.7
-install:
```

Comment:
Why'd you delete all this

Reply:
@neuromusic  was downloading conda and installing and running everything in a virtual environment which worked, but was unnecessary. Also he was explicitly installing each package in the setup instead of having a requirements.txt which is what this PR is for.